src/components/fieldText/fieldText.tsx

```diff
@@ -60,6 +61,7 @@ export const FieldText: FC<FieldTextProps> = forwardRef(
       hasDoubleMessage = false,
       type = 'text',
       displayError = true,
+      maxLength = 256,
```

Comment:
Let's not set a default value.
Because there may be cases where you don't need to cut the string, but show an error.